bootstrap.json

```diff
@@ -1,7 +1,6 @@
 {
     "after-scripts": [
-        "brew linkapps",
```

Comment:
its still there.. see below. Github's diff checker is not very smart. it did not detect that i removed the comma at the end of the line.